src/plugins/visualizations/public/utils/saved_visualize_utils.test.ts

```diff
@@ -43,10 +43,10 @@ jest.mock('../../../../plugins/data/public', () => ({
 }));
 
 const mockInjectReferences = jest.fn();
-const mockExtractReferences = jest.fn(() => ({ references: [], attributes: {} }));
+const mockExtractReferences = jest.fn((arg) => arg);
```

Comment:
why removing the reference object here rather than apple the `arg` on top of them?

Reply:
The `saveVisualization` function [always calls](https://github.com/andrewctate/kibana/blob/04c13085295fdd80cdf2fdf90601fa18d8a0331d/src/plugins/visualizations/public/utils/saved_visualize_utils.ts#L365-L366) `extractReferences` with an object of that structure, making the reference object redundant. Defining our own mock object also decreases how realistic these tests are, since someone could break that call in `saveVisualization` and any tests that rely on the reference object would still pass.

For those reasons, I didn't see the benefit of defining our own mock object.

That said, it looks like these changes are holdovers from an earlier version of the PR where I required the change for a new test. So, we can revert if you think this is a mistake!